test/es-module/test-typescript.mjs

```diff
@@ -28,6 +28,18 @@ test('execute a TypeScript file with imports', async () => {
   strictEqual(result.code, 0);
 });
 
+test('execute a TypeScript file with imports', async () => {
+  const result = await spawnPromisified(process.execPath, [
+    '--no-warnings',
+    '--eval',
+    `assert.throws(() => require(${JSON.stringify(fixtures.path('typescript/ts/test-import-fs.ts'))}), {code: 'ERR_REQUIRE_ESM'})`,
```

Comment:
```suggestion
    `require("assert").throws(() => require(${JSON.stringify(fixtures.path('typescript/ts/test-import-fs.ts'))}), {code: 'ERR_REQUIRE_ESM'})`,
```

Reply:
Why?

Reply:
Oh! I didn't know assert was global in the eval.